boa3/internal/model/builtin/contract/nep17_interface_methods/nep17interfacemethod.py

```diff
@@ -0,0 +1,45 @@
+import ast
+from typing import Dict, Optional, List
+
+from boa3.internal.compiler.compiledmetadata import CompiledMetadata
+from boa3.internal.model.builtin.interop.contract import ContractType
+from boa3.internal.model.builtin.method.builtinmethod import IBuiltinMethod
+from boa3.internal.model.type.itype import IType
+from boa3.internal.model.variable import Variable
+
+
+class Nep17InterfaceMethod(IBuiltinMethod):
+
+    def __init__(self, args: Dict[str, Variable], identifier: str,
+                 native_identifier: str, return_type: IType, defaults: List[ast.AST] = None):
+        super().__init__(identifier, args, return_type=return_type, defaults=defaults)
+        self.native_identifier = native_identifier
```

Comment:
`native_identifier`  could be an optional argument, if not set then it's the same as `identifier`
 ```suggestion
    def __init__(self, args: Dict[str, Variable], identifier: str,
                 return_type: IType, defaults: List[ast.AST] = None,
                 native_identifier: str = None):
        super().__init__(identifier, args, return_type=return_type, defaults=defaults)
        if not native_identifier:
            native_identifier = identifier
        self.native_identifier = native_identifier
```

this would make it easier to read what the arguments should mean in the inherited classes.